### PR TITLE
Update the `appVersion` in the seqr Helm Chart when there is a push the branches [dev, master]

### PR DESCRIPTION
**Pull request by ShifaSZ**
The newly added Github workflow creates a PR in the [seqr-helm repo](https://github.com/broadinstitute/seqr-helm) to update the `appVersion` in the seqr Helm Chart when there is a push the release branch. Here is an example of the PR: https://github.com/broadinstitute/seqr-helm/pull/9

### CODE REVIEW

File: .github/workflows/update-seqr-helm-version.yml
Comment (ShifaSZ) [inline .github/workflows/update-seqr-helm-version.yml:5]: It needs to be updated to `master` eventually.

File: .github/workflows/update-seqr-helm-version.yml
Comment (ShifaSZ) [inline .github/workflows/update-seqr-helm-version.yml:25]: Should we push to the main branch directly rather than creating a PR so that it is fully automated?

File: .github/workflows/update-seqr-helm-version.yml
Comment (sjahl) [inline .github/workflows/update-seqr-helm-version.yml:15]: This is cool, I didn't know you could add items to the runner's environment variables like this!

File: .github/workflows/update-seqr-helm-version.yml
Comment (sjahl) [reply .github/workflows/update-seqr-helm-version.yml:25]: Yes, I think pushing to main (the default branch is "main" rather than "master" on this repo) is probably the right thing to do. If we had a fully automated rollout to prod, then we'd probably want a PR, but I think the plan is to have production pin to a specific commit of the helm chart, so it should be fine to push right to main for this.

**Review by sjahl — APPROVED** [review]

**Comment by sjahl** [comment]
Looks good to me so far, I can re-review once the suggestion to push to main rather than opening a PR is done

**Review by sjahl — COMMENTED** [review]
This all looks pretty good! I have some work to do with regards to automating updates to the dev environment, and I'll need to use this branch/PR for testing before we merge. I'll take this one off your hands @ShifaSZ

**Comment by hanars** [comment]
@sjahl should we merge this?

**Comment by sjahl** [comment]
No, we should probably close it actually. I'm going to use some of what Shifa put together, but it's going to end up as part of different github workflow configurations

**Pull request closed without merging**
